great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -149,7 +153,16 @@ def auth_headers(self) -> Dict[str, str]:
     def _get(self, key: Tuple[str, ...]) -> dict:
         ge_cloud_url = self.get_url_for_key(key=key)
         try:
-            response = requests.get(ge_cloud_url, headers=self.auth_headers)
+            params: dict
+            # if name is included in the key, add as a param
+            if len(key) > 2 and key[2]:
```

Comment:
In the current architecture, the key here has already been converted into a tuple hence the checking for length. It would be better here to use an identifier type but saving that for a future refactor effort since it may not be trivial and touch several store types.

Reply:
ugh yeah I tried this refactor some time ago and it's tough - we use `key.to_tuple()` for all stores as implemented in `store.py`.

I don't fully remember why we do this but it results in us losing strong typing.